components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -197,7 +197,13 @@ public int getOptimalTileHeight() {
   public boolean isThisType(RandomAccessInputStream stream) throws IOException {
     final int blockLen = 1;
     if (!FormatTools.validStream(stream, blockLen, true)) return false;
-    return stream.read() == LIF_MAGIC_BYTE;
+    if (stream.read() != LIF_MAGIC_BYTE) return false;
+    stream.skipBytes(7);
+    if (stream.readByte() != LIF_MEMORY_BYTE) return false;
+    int nc = stream.readInt();
+    String desc = DataTools.stripString(stream.readString(nc * 2));
+    if (desc.equals("LMS_Object_File")) return false; //LOF file
+    return true;
   }
```

Comment:
I changed this function since LIF and LOF files start with the same magic numbers, so the distinction has to be done via the existence of "LMS_Object_File" in the header